woocommerce/woocommerce-admin
# Onboarding: Redesign plugin benefits screen

Fixes #3709 

Redesign the benefits screen to match the new mockups.

@jameskoster Can you give this a quick review- specifically smaller screen sizes?

### Screenshots
<img width="1064" alt="Screen Shot 2020-02-24 at 1 50 20 PM" src="https://user-images.githubusercontent.com/10561050/75154451-f3851d00-570d-11ea-874e-87a3276901f5.png">

### Detailed test instructions:

1. Uninstall Jetpack and WooCommerce services.
1. Walk through the profiler.
1. Make sure the benefits (after the theme step) screen matches the design.
1. Make sure install still works.
1. Try deactivating WCS or Jetpack and re-enable the profiler.
1. Make sure the benefits and plugin names change according to what's not installed and activated.
